README.md

```diff
@@ -38,6 +38,17 @@ It is **strongly** recommend to use Docker. See instructions above.
 
 Access the app via `localhost:5000`
 
+# About Stocks in the Future 
+
+## Mission
+[Stocks in the Future](https://sifonline.org/) is a program with the mission of developing highly motivated middle school students who are eager to learn and dedicated to attending class through the use of incentives coupled with a financial literacy curriculum focused on investing that reinforces Math, Language Arts and Social Studies. Stocks in the Future pushes to educate, encourage, and empower the next generation of financially-literate individuals.
+
+## Ruby for Good
+If you have any questions about an issue, comment on the issue, open a new issue, or ask in [the RubyForGood slack] (https://join.slack.com/t/rubyforgood/shared_invite/zt-2k5ezv241-Ia2Iac3amxDS8CuhOr69ZA). Stocks-in-the-future has a #stocks-in-the-future channel in the Slack. Feel free to join the community! 
+
+## Repo Deployment
+Check out this link for information about the deployment of this repo: https://stocks-in-the-future-2e5bd264d30e.herokuapp.com/
```

Comment:
Maybe add something about who to ask for access to this?

Reply:
Hi Liz, could you elaborate on what you mean? Access to what?

Reply:
When I click on the link for information about deployment, I don't see anything, I assume because I don't have permission to access it? I'm thinking if most people won't have the ability to access it, maybe it doesn't belong in the read me, unless we want to let people know who they would need to ask.

Reply:
Ohh I see. I put the link in the ReadMe because the issue said that it was an expectation to be able to complete the task (reference issue #45). I'm new to contributing so I'm not entirely sure what the issue with access is. If you know then please let me know, if not then I think I'll have to wait to see if someone else has an idea.

Reply:
That's a rather old issue. Still relevant, but I think we can leave out the deployment part. Very few contributors will need that information. Any objections from the other leads? @abachman @h-m-m

Reply:
Ok I should be able to go ahead and take that part out.

Reply:
Hi there! You should now be able to see a new PR without the broken link. Thanks